src/test/java/org/cloudfoundry/promregator/cfaccessor/CFAccessorCacheCaffeineTimeoutSpringApplication.java

```diff
@@ -20,21 +20,9 @@ public InternalMetrics internalMetrics() {
 		return Mockito.mock(InternalMetrics.class);
 	}
 	
-	private static class TimeoutMonoAnswer implements Answer<Mono<?>> {
-		@Override
-		public Mono<?> answer(InvocationOnMock invocation) throws Throwable {
-			return Mono.error(new TimeoutException("Unit test timeout raised"));
-		}
-	}
-	
 	@Bean
 	public CFAccessor parentMock() {
-		CFAccessor mock = Mockito.mock(CFAccessor.class);
-		Mockito.when(mock.retrieveOrgId("dummy")).then(new TimeoutMonoAnswer());
-		Mockito.when(mock.retrieveSpaceId("dummy1", "dummy2")).then(new TimeoutMonoAnswer());
-		Mockito.when(mock.retrieveAllApplicationIdsInSpace("dummy1", "dummy2")).then(new TimeoutMonoAnswer());
-		Mockito.when(mock.retrieveSpaceSummary("dummy")).then(new TimeoutMonoAnswer());
```

Comment:
Moved the mocking calls into the `@BeforeEach` method of the test so verifications are reset properly between tests.